test/dummy/app/tasks/maintenance/update_posts_throttled_task.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+module Maintenance
+  class UpdatePostsThrottledTask < MaintenanceTasks::Task
+    class << self
+      attr_accessor :throttle
```

Comment:
IMHO this is useful mostly just for our tests. If we think we'd like a throttling task for local development in the future, we should probably write something with a more realistic throttling condition. I'm going to leave this for now because I'm not sure it warrants the time investment yet 😄